src/imports/logger/ErrorsTransport.js

```diff
@@ -34,23 +33,29 @@ export default class ErrorsTransport extends Transport {
     }
   }
 
+  createError({ name, message, code, stack }) {
```

Comment:
please do as i and @sirpy asked:
- modify message
- send to Sentry
- await once sent
- restore message